src/main/java/com/minecolonies/coremod/blocks/cactus/BlockCactusTrapdoor.java

```diff
@@ -0,0 +1,21 @@
+package com.minecolonies.coremod.blocks.cactus;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.blocks.AbstractBlockTrapdoor;
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+
+import java.util.Locale;
+
+public class BlockCactusTrapdoor extends AbstractBlockTrapdoor<BlockCactusTrapdoor> {
+
+    public BlockCactusTrapdoor() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)